### DISCUSSION TITLE
Compilation error sys/scoket.h

### DISCUSSION THREAD

**Discussion by Selmacas** [bozimmerman/Zimodem#105] [Q&A]
Hello,
could I ask you, I have arduino ide v 1.8.10 a installed board esp8266 version 2.7.4. When I try to build this project (version 4.0.0) it ends with error:

```
 In file included from /tmp/arduino_build_562253/sketch/src/libssh2/bcrypt_pbkdf.c:21:0:
/tmp/arduino_build_562253/sketch/src/libssh2/libssh2_priv.h:108:25: fatal error: sys/socket.h: No such file or directory
 # include <sys/socket.h>
                         ^
compilation terminated.
Použití knihovny ESP8266
```

How could I solve this problem.
Thank you for your answer

**Comment by andromeda2911** [comment]
I have the same problem and I never had an answer and no information on the internet forums
